suite-native/app/src/App.tsx

```diff
@@ -22,11 +22,16 @@ import { applicationInit } from './initActions';
 import { useReportAppInitToAnalytics } from './hooks/useReportAppInitToAnalytics';
 import { SentryProvider } from './SentryProvider';
 import { ModalsRenderer } from './ModalsRenderer';
+import { envSchema } from '../envValidation';
 
 if (__DEV__) {
     require('./LogBox');
 }
 
+// process.env validation. This will fail if environment variables defined in schema
+// are not passed to the application from the runner.
+envSchema.parse(process.env);
```

Comment:
this won't work, in production object will be mostly empty because env varaibles are replaced not injected into process.env. Aka `process.env.HELLO` => `"Hello world variable content"` but `Object.keys(process.env)` => `[]`

Reply:
It will work in debug because env variables works differently there